src/lib/homestore_backend/hs_blob_manager.cpp

```diff
@@ -409,7 +415,12 @@ void HSHomeObject::on_blob_del_commit(int64_t lsn, sisl::blob const& header, sis
     }
 
     auto& multiBlks = r.value();
-    if (multiBlks != tombstone_pbas) { repl_dev->async_free_blks(lsn, multiBlks); }
+    if (multiBlks != tombstone_pbas) {
+        repl_dev->async_free_blks(lsn, multiBlks);
+        hs_pg->durable_entities_.active_blob_count.fetch_sub(1, std::memory_order_relaxed);
```

Comment:
Does it make sense to make these three lines in a single function, say update_durable_ent() and inside this funciton hs_pg is set to dirty? The point is to give api to give a hint to others that this should not be accidentally updated without updating the dirty flag.

Reply:
I have protected the access in a separate method and allow updates to happen later.